packages/babel-preset-es2015/index.js

```diff
@@ -1,25 +1,70 @@
-module.exports = {
-  plugins: [
-    require("babel-plugin-transform-es2015-template-literals"),
-    require("babel-plugin-transform-es2015-literals"),
-    require("babel-plugin-transform-es2015-function-name"),
-    require("babel-plugin-transform-es2015-arrow-functions"),
-    require("babel-plugin-transform-es2015-block-scoped-functions"),
-    require("babel-plugin-transform-es2015-classes"),
-    require("babel-plugin-transform-es2015-object-super"),
-    require("babel-plugin-transform-es2015-shorthand-properties"),
-    require("babel-plugin-transform-es2015-duplicate-keys"),
-    require("babel-plugin-transform-es2015-computed-properties"),
-    require("babel-plugin-transform-es2015-for-of"),
-    require("babel-plugin-transform-es2015-sticky-regex"),
-    require("babel-plugin-transform-es2015-unicode-regex"),
-    require("babel-plugin-check-es2015-constants"),
-    require("babel-plugin-transform-es2015-spread"),
-    require("babel-plugin-transform-es2015-parameters"),
-    require("babel-plugin-transform-es2015-destructuring"),
-    require("babel-plugin-transform-es2015-block-scoping"),
-    require("babel-plugin-transform-es2015-typeof-symbol"),
-    require("babel-plugin-transform-es2015-modules-commonjs"),
-    [require("babel-plugin-transform-regenerator"), { async: false, asyncGenerators: false }],
-  ]
+module.exports = function(context, opts) {
+  /* eslint-disable no-var */
```

Comment:
@hzoo I just updated this PR. I added these since this was failing linting, but this file isn't transpiled at the moment, they are only published as-is.